cmd/thanos/query.go

```diff
@@ -70,6 +70,8 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 	maxConcurrentQueries := cmd.Flag("query.max-concurrent", "Maximum number of queries processed concurrently by query node.").
 		Default("20").Int()
 
+	lookbackDelta := cmd.Flag("query.lookback-delta", "The maximum lookback duration for retrieving metrics during expression evaluations.").Duration()
```

Comment:
Can we describe it better? It's not maximum but actually lookback delta right? And also we should explain what it means for PromQL or link to docs :hugs:

Reply:
Added some more details, let me know what you think.

Unfortunately, there are official Prom docs about this flag.